lists/gr.csv

```diff
@@ -38,3 +38,8 @@ https://www.migrant.gr/,HUMR,Human Rights Issues,2018-09-20,OONI,
 http://www.efsyn.gr/,NEWS,News Media,2018-09-20,OONI,
 http://astegoi.gr/,HUMR,Human Rights Issues,2018-09-20,OONI,
 https://www.amnesty.gr/,HUMR,Human Rights Issues,2018-09-20,OONI,
+http://www.subtitles.gr,SRCH,Search Engines,2018-11-09,Anonymous,
+http://www.subztv.club/,CULTR,Culture,2018-11-09,Anonymous,
```

Comment:
I believe these two should be categorised as FILE.

Reply:
```
File-sharing 	FILE 	P2P SFTWR WRZ 	Sites and tools used to share files, including cloud-based file storage, torrents and P2P file-sharing tools.
```
I don't think that they fit into the FILE category, these sites are not torrent p2p or cloud-based file storage sites, if we want to add them into this category perhaps we should also update the category description.